release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -140,6 +140,11 @@ For more information, see xref:../installing/installing_bare_metal_ipi/ipi-insta
 
 Using {op-system-base-full} 9 with the OpenShift CLI (`oc`) is now supported.
 
+[NOTE]
+====
+It is not supported to install the OpenShift CLI (`oc`) as an RPM for {op-system-base-full} 9. You must install the OpenShift CLI for {op-system-base} 9 by downloading the binary.
+====
+
```

Comment:
Just a suggestion: do you think it would be helpful to add a "for more information" and link to the steps you have in your other PR?

Reply:
Updated to add a link to installing the CLI!